### PR TITLE
Fix reporting of totals

### PR DESCRIPTION
**Pull request by zhrebicek**
We were calculation `_rate` which we called `_total`.
First simplistic shot is to add global accumulator for total per key with labels.

### CODE REVIEW

File: lib/coralogix.js [hunk 1]
```diff
@@ -37,6 +37,7 @@ let coralogixStats = {};
 let coralogixPrefix;
 let coralogixMappings = [];
 let coralogixConfig = {};
+let totalsAccumulator = new Map()
```
Comment (rtimush) [inline lib/coralogix.js:42]: Ideally, we should have some TTL for the values in this map, otherwise it will grow infinitely.

**Review by rtimush — APPROVED** [review]
Looks good to me

**Comment by zhrebicek** [comment]
I see that I am ignoring that counters like this are same, but our acumulator would not think so, would fix tomorrow:
```
timer_asdasdtest:10|c|#tag_1:value,tag_2:value_2
timer_asdasdtest:10|c|#tag_2:value_2,tag_1:value
```

File: README.md [hunk 2]
```diff
@@ -9,6 +9,11 @@ A plugin to connect StatsD to [Coralogix].
 
 Coralogix documentation can be found [here](https://coralogix.com/docs/statsd/).
 
+## Breaking change notice
+
+* In version `0.1.*` we were calculating `_total` for counters as rate per sedond. This was fixed in `0.2.0` where we
```
Comment (kolov) [inline README.md]: sedond => second

File: lib/coralogix.js [hunk 3]
```diff
@@ -216,44 +229,57 @@ const flush_stats = function coralogix_post_stats(ts, metrics) {
...
 
             // TODO support for quantiles
         }
     }
 
     post_stats(payload);
+    pruneTotalsAccumularot(ts);
 };
 
+function pruneTotalsAccumularot(currentTs){
```
Comment (kolov) [inline lib/coralogix.js]: acumulatorot?

File: lib/coralogix.js [hunk 4]
```diff
@@ -216,44 +229,57 @@ const flush_stats = function coralogix_post_stats(ts, metrics) {
...
+    totalsAccumulator.forEach( (value, key, _map) => {
+        let [_value,tsLiveUntil] = value;
+        if (currentTs >= tsLiveUntil){
+            console.log("Deleting key",key,(tsLiveUntil - currentTs))
```
Comment (kolov) [inline lib/coralogix.js]: one log for every point - maybe too much?

File: README.md [hunk 5]
```diff
@@ -9,6 +9,11 @@ A plugin to connect StatsD to [Coralogix].
 
 Coralogix documentation can be found [here](https://coralogix.com/docs/statsd/).
 
+## Breaking change notice
+
+* In version `0.1.*` we were calculating `_total` for counters as rate per sedonds. This was fixed in `0.2.0` where we
+  added internal acumulator for `_total` of counters and we do report the metric correctly.
```
Comment (rtimush) [inline README.md:15]:
```suggestion
  added internal accumulator for `_total` of counters and we do report the metric correctly.
```

File: lib/coralogix.js [hunk 6]
```diff
@@ -280,11 +306,11 @@ exports.init = function coralogix_init(startup_time, config, events, log) {
     coralogixApiHost = coralogixConfig.apiHost;
     coralogixPrefix = coralogixConfig.prefix;
     coralogixMappings = new Map(Object.entries(coralogixConfig.mappings));
+    ttlSeconds = coralogixConfig.totalsAccumulatorTtlSeconds ?? 3600;
```
Comment (kolov) [inline lib/coralogix.js:308]: My Javascript is not strong - I don't know ??.  || would probably work too.

Comment (kolov) [reply lib/coralogix.js:308]: Just googled it, looks fine :-)

File: lib/coralogix.js (hunk 4 above)
Comment (zhrebicek) [reply lib/coralogix.js]: This should have been deleted.

File: lib/coralogix.js (hunk 6 above)
Comment (zhrebicek) [reply lib/coralogix.js:308]: Yeah it is https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing `||` does different thing.

**Review by rtimush — APPROVED** [review]

**Pull request merged**